### PR TITLE
JBPM-7773: Update Process Instance logs screen to work with dataset q…

### PR DESCRIPTION
**Pull request by nmirasch**
…ueries

### CODE REVIEW

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac replaced the ProcessRuntimeDataService call by a new dataset query. could you take a look? I retrieve there all processinstancelogs field, maybe it doesn't make sense and we can limit the retrieved fields to the ones what are used by the UI.  Could you take a look, please?
Thanks

**Comment by nmirasch** [comment]
@cristianonicolai @bsremac Added suggested changes and Renamed  any reference to  RuntimeLog* to ProcessInstanceLog* .

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RuntimeLogSummaryMapper.java [hunk 1]
```diff
@@ -17,21 +17,21 @@
 
 import java.util.function.Function;
 
-import org.jbpm.workbench.pr.model.RuntimeLogSummary;
+import org.jbpm.workbench.pr.model.ProcessInstanceLogSummary;
 import org.kie.server.api.model.instance.NodeInstance;
 
-public class RuntimeLogSummaryMapper implements Function<NodeInstance, RuntimeLogSummary> {
+public class RuntimeLogSummaryMapper implements Function<NodeInstance, ProcessInstanceLogSummary> {
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RuntimeLogSummaryMapper.java]: I guess this one can also be removed?

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogPresenter.java [hunk 2]
```diff
@@ -147,25 +193,27 @@ public void refreshProcessInstanceData(final LogOrder logOrder,
...
+
+        dataSetQueryHelper.setCurrentTableSettings(filterSettingsManager.createDefaultFilterSettingsPrototype(event.getProcessInstanceId()));
         refreshProcessInstanceData(LogOrder.ASC,
                                    LogType.BUSINESS);
     }
 
-    @Inject
-    public void setProcessRuntimeDataService(final Caller<ProcessRuntimeDataService> processRuntimeDataService) {
-        this.processRuntimeDataService = processRuntimeDataService;
+    protected ProcessInstanceLogSummary createProcessInstanceLogSummaryFromDataSet(DataSet dataSet,
```
Comment (cristianonicolai) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogPresenter.java]: There is probably not much value in keeping this method now.

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RuntimeLogSummaryMapper.java (hunk 1 above)
Comment (nmirasch) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RuntimeLogSummaryMapper.java]: ups, true!

**Comment by nmirasch** [comment]
@cristianonicolai done :)

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Pull request merged**